image/templates/helm/stackrox-secured-cluster/internal/defaults/50-images.yaml.htpl

```diff
@@ -13,11 +13,19 @@ image:
   scanner:
     name: [< required "" .ScannerSlimImageRemote >]
     tag: [< required "" .ScannerImageTag >]
-  [< if (not .KubectlOutput) >]
+  [<- if not .KubectlOutput >]
   scannerDb:
     name: [< required "" .ScannerDBSlimImageRemote >]
     tag: [< required "" .ScannerImageTag >]
-  [< end >]
+  [<- end >]
+  [<- if and (not .KubectlOutput) .FeatureFlags.ROX_SCANNER_V4 >]
+  scannerV4:
+    name: [< required "" .ScannerV4ImageRemote >]
+    tag: [< required "" .ScannerV4ImageTag >]
+  scannerV4DB:
+    name: [< required "" .ScannerV4DBImageRemote >]
+    tag: [< required "" .ScannerV4ImageTag >]
```

Comment:
I don't recall is there a `.ScannerV4DBImageTag` or no?

Reply:
No, there isn't.